crypto/asn1/x_bignum.c

```diff
@@ -130,9 +130,16 @@ static int bn_c2i(ASN1_VALUE **pval, const unsigned char *cont, int len,
 static int bn_secure_c2i(ASN1_VALUE **pval, const unsigned char *cont, int len,
                          int utype, char *free_cont, const ASN1_ITEM *it)
 {
+    int ret;
+    BIGNUM *bn;
+
     if (!*pval)
         bn_secure_new(pval, it);
```

Comment:
and, while you are already at it, could you please also check the return value of bn_secure_new ?

Reply:
Sure, no problem.

Reply:
Should'a used "!= NULL" in the test.